.github/workflows/api.yml

```diff
@@ -18,4 +18,4 @@ jobs:
     - name: Upload coverage reports to Codecov with GitHub Action
       uses: codecov/codecov-action@v4.2.0
       env:
-      	CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
The error: https://github.com/sdzhong/codecov-demo/actions/runs/8794132057/workflow 

![image](https://github.com/sdzhong/codecov-demo/assets/89550162/51b80cf3-8ad9-499e-a5b9-74fac363e65e)


The root cause was a bad code snippet: I copy-pasted from https://docs.codecov.com/docs/github-4b-getting-aggregate-coverage > `.github/workflows/api.yml` code snippet

![image](https://github.com/sdzhong/codecov-demo/assets/89550162/5eeb9b18-1e02-45cf-a410-fc1509eff34d)